src/main/java/de/craften/plugins/rpgplus/scripting/api/entities/events/EntityEventManagerImpl.java

```diff
@@ -33,6 +33,10 @@ public EntityEventManagerImpl(SafeInvoker invoker) {
     }
 
     private Multimap<String, LuaFunction> getHandlers(Entity entity) {
+    	if (entity == null) {
+    		return null;
```

Comment:
When will `entity` be `null` here? Maybe we should throw an `IllegalArgumentException` if that happens?